geode-core/src/main/java/org/apache/geode/cache/execute/FunctionContext.java

```diff
@@ -98,4 +99,11 @@ default String getMemberName() {
 
     return member.getId();
   }
+
+  /**
+   * A convenience method to get the distributed system
+   *
+   * @return Returns the distributed system
+   */
+  DistributedSystem getDistributedSystem();
```

Comment:
I don't think we need to explicitly expose a method for `getDistributedSystem` as this field can be retrieved off the cache.
Nor do I see a reason to change the public API.